test/docker-compose-e2e.yaml

```diff
@@ -6,7 +6,7 @@ services:
       - 4222:4222
   # Have hosts in 3 different "regions"
   wasmcloud_east:
-    image: wasmcloud/wasmcloud:0.79.0
+    image: wasmcloud/wasmcloud:0.81.0
```

Comment:
Is it worth testing a mix of 0.81 and 0.82 hosts for coverage of both heartbeat payload structures simultaneously?

Reply:
Oh, it might be, but 0.82 doesn't exist yet 🙃